spec/factories/organizations.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :organization do
+    name Faker::Company.name
+    organization_form FactoryBot.create(:organization_form)
```

Comment:
you can use `association` here.

otherwise you need to use block in case you use `organization_form`. this will allow each organization to have its own `organization_form`.

Read more here:
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations